src/transformers/configuration_utils.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, **kwargs):
         # Attributes with defaults
         self.output_attentions = kwargs.pop("output_attentions", False)
         self.output_hidden_states = kwargs.pop("output_hidden_states", False)
-        self.output_past = kwargs.pop("output_past", True)  # Not used by all models
+        self.use_cache = kwargs.pop("use_cache", True)  # Not used by all models
```

Comment:
still needed on config?

Reply:
That's a good question - I added it because all inputs of `generate()` are on config and it allows `task_specific` params to overwrite the default model params.